### ISSUE TITLE
Please support using jsdelivr.com CDN

### ISSUE THREAD

**Issue by boustanihani** [shoelace-style/webawesome#1468]
Please support using jsdelivr:
https://www.jsdelivr.com/package/npm/@awesome.me/webawesome

The default suggested code for installing does not work:

<img width="1030" height="436" alt="Image" src="https://github.com/user-attachments/assets/433ff28f-d49a-453e-a9af-04105fd62c3c" />

`Latest Minor` also does not work:

<img width="1030" height="436" alt="Image" src="https://github.com/user-attachments/assets/72d5ebf6-c2c1-4577-afa6-8e232bf68f70" />

Also check this warning: `No default CSS file set by the package author so the URL is guessed. You can always browse all package files to use another one.`

This works:

```
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@awesome.me/webawesome@3.0.0-beta.6/dist-cdn/styles/webawesome.css">

<script type="module" src="https://cdn.jsdelivr.net/npm/@awesome.me/webawesome@3.0.0-beta.6/dist-cdn/webawesome.loader.js"></script>
```
